lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -103,6 +103,7 @@ function dosomething_reportback_form($form, &$form_state, $entity = NULL) {
   else {
     // Output reportback images with date last updated.
     $form['reportback_submissions']['reportback_item']['#attributes']['is_supersized'] = FALSE;
+    $form['reportback_submissions']['reportback_item']['#attributes']['data-validate'] = '';
```

Comment:
normally we would just `unset` these values, but I think we still want to validate the item on update, just don't require an image, right?